### DISCUSSION TITLE
Avoid errors in posts widget aria-label on non english pages

### DISCUSSION THREAD

**Discussion by Speaka** [elementor/elementor#24584] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Adding customs aria-labels to read-more links in posts widget.
On non english installations you will get lighthouse errors and seo penalties if the linktext and aria-label mismatch.
For example in german you'll translate read more to weiterlesen for native experience. This produces this html:
```
<a class="elementor-post__read-more" href="example.com/post123" aria-label="Read more about here comes the post title" tabindex="-1">
Weiterlesen »
</a>
```
But this results in the following lighthouse error (and penalties in other seo/accesibility tools):
> [!WARNING]
> Elements with visible text labels do not have matching accessible names.
> Visible text labels that do not match the accessible name can result in a confusing experience for screen reader users.

### Describe the solution you'd like.

To resolve this issue it needs to either translate the aria-label prefix "Read more about " to the installations language oder enable a custom string.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by heckpet** [comment]
+1 - Same on my site. Please fix this! Not sure, if this should be an issue instead of a feature request...
